integrations/jenkins/Jenkinsfile

```diff
@@ -777,7 +783,9 @@ pipeline {
                             ORT_OPTIONS="$ORT_OPTIONS --stacktrace"
                         fi
 
-                        ADVISOR_OPTIONS="-a OSV,VulnerableCode"
+                        if [ -n "ADVISORS" ]; then
+                            ADVISOR_OPTIONS="-a $ADVISORS"
```

Comment:
Should $ADVISORS be put in quotes? (e.g. it might tolerate `", "`)

Reply:
> Should $ADVISORS be put in quotes? (e.g. it might tolerate `", "`).

We don't tolerate `", "` in similar existing code, like the reporters, either, so I'd like to align on that behavior for now.

> Does it actually need to be executed only conditionally?

I believe yes, because if someone clears the list of advisors from the parameter field (to enforce ORT's built-in default behavior), then we won't want to add `-a`.